pages/admin/courses/course_list.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div id="page">
+  <v-container class="layout" v-if="!flagView"></v-container>
+  <div v-else>
+      <header class="title">
+        <h1>MEUS CURSOS</h1>
+        <v-btn class="mx-2 btn-icon" icon>
+          <v-icon dark id="plus-icon">mdi-plus-circle</v-icon>
+        </v-btn>
+      </header>
+      <div class="body-list">
+        <v-card v-for="item in list" v-bind:key="item.id" class="v-card-border">
+          <div class="content">
+            <div class="img-mask">
+              <img :src="item.thumbUrl" alt />
+            </div>
+            <div class="info-text">
+              <h1>{{item.title}}</h1>
+              <p>{{item.description}}</p>
+            </div>
+          </div>
+          <div class="group-buttons">
+            <v-btn class="btn-item bg-blue">
+              <v-icon class="text-white">mdi-border-color</v-icon>
+            </v-btn>
+            <v-btn class="btn-item bg-danger" @click="deleteCourse(item.id)">
+              <v-icon class="text-white">mdi-delete</v-icon>
+            </v-btn>
+          </div>
+        </v-card>
+      </div>
+      <navigation-bar />
+    </div>
+  </div>
+</template>
+<router>
+{
+    path: '/admin/listar-cursos'
+  }
+</router>
+<script>
+import NavigationBar from '~/components/NavigationBar.vue'
+import http from '~/services/http/generic'
+export default {
+  data: () => ({
+    list: [],
+    flagView : false
+  }),
+  components: {
+    NavigationBar,
+  },
+  methods: {
+    getAllCourses() {
```

Comment:
```suggestion
    loadAllCourses() {
```
Esse método não retorna os cursos, ele carrega os dados apenas, então sugiro alterar de get para load

Reply:
Então, esse método faz uma chamada para um outro serviço genérico que faz o retorno dos cursos, de qualquer forma a função que é chamada apenas faz a execução da chamada http retornando uma lista de curso, que é armazenada no data, é uma boa dica porém não acho impeditivo, eu não mudaria, digo isso pra deixarmos o dev pegar um pouco de maturidade e responsabilidade na hora de criar métodos.